set_env.sh

```diff
@@ -0,0 +1,7 @@
+export SNORKELHOME="$HOME/Documents/snorkel"
```

Comment:
Short answer: They are necessary to run snorkel.

Long answer: The way snorkel is set up is that they are setting the snorkel path as a global variable. They do this in order to set up the python path for module inclusion/checking to see if stanford's corenlp is installed. I don't know a way around it.